corehq/apps/data_cleaning/templates/data_cleaning/partials/offcanvas.html

```diff
@@ -16,7 +16,14 @@ <h5 class="offcanvas-title">
     ></button>
   </div>
   <div class="offcanvas-body">
-    <p>todo</p>
+    {% for filter_type in pinned_filter_types %}
+      <div
+        hx-get="{% url "data_cleaning_pinned_filter_form" domain session_id filter_type %}"
```

Comment:
It seems like the `domain` and `session_id` is not accessed or used in the `PinnedFilterFormView`... or did I miss something?

Reply:
it certainly is! `DomainViewMixin` uses `domain`, `BulkEditSessionViewMixin` uses `session_id` and ultimately creates a `session`. It's not used in the `PinnedFilterFormView` via `get_context_data` right now because it's currently a placeholder view. However, it certainly will be used!